pkgs/by-name/op/opengist/package.nix

```diff
@@ -11,25 +11,17 @@
 let
   # finalAttrs when 🥺 (buildGoModule does not support them)
   # https://github.com/NixOS/nixpkgs/issues/273815
-  version = "1.8.3";
+  version = "1.8.4";
   src = fetchFromGitHub {
     owner = "thomiceli";
     repo = "opengist";
     rev = "v${version}";
```

Comment:
```suggestion
    tag = "v${version}";
```
If you don't mind, please switch to the new `tag` syntax.
Also, change the `meta.changelog` to `"https://github.com/thomiceli/opengist/releases/tag/${src.tag}"`.

Reply:
changelog.md is fine, no? it is up to date.

Reply:
Sorry, I meant `"https://github.com/thomiceli/opengist/blob/${src.tag}/CHANGELOG.md"`

Reply:
done